### DISCUSSION TITLE
How to use the plugin's js-behavior options?

### DISCUSSION THREAD

**Discussion by nicevlhjfihtqngkldl** [htmlstreamofficial/preline#434] [Q&A] [answered]
https://preline.co/plugins/html/dropdown.html#js-behavior
![image](https://github.com/user-attachments/assets/cff4df87-0d8a-4463-bf01-edb3284db91d)
How to apply these options? Could someone give me a code example. 
Thanks very much.

**Comment by langscot** [comment, marked as answer]
This should be documented better, but whenever you see the `[--xxxxx]` format, these are intended to be CSS styles. As an example, you can do the following:

```html
<div style="--placement: top-left;"></div>
```

However, assuming you are using TailwindCSS, you can use [Tailwind's shorthand notation](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-properties) for using arbitrary CSS properties, like so:

```html
<div class="[--placement:top-left]"></div>
```

The documentation uses Tailwind's shorthand notation, so whenever you see an option wrapped in `[]` it is likely to be applied this way. Hope it helps.
